package/kedro_viz/launchers/cli/run.py

```diff
@@ -123,16 +123,21 @@ def run(
     )
     from kedro_viz.server import run_server
 
-    kedro_project_path = _find_kedro_project(Path.cwd())
-
-    if kedro_project_path is None:
-        display_cli_message(
-            "ERROR: Failed to start Kedro-Viz : "
-            "Could not find the project configuration "
-            f"file '{_PYPROJECT}' at '{Path.cwd()}'. ",
-            "red",
-        )
-        return
+    kedro_project_path = None
```

Comment:
maybe this `kedro_project_path` does not need to be defined outside the if-else statement if it is only used within the else block?

Reply:
it is needed later, it is passed to run_server